apps/meteor/ee/app/livechat-enterprise/server/api/outbound.ts

```diff
@@ -0,0 +1,35 @@
+import { API } from '../../../../../app/api/server';
+import { isGETOutboundProviderParams } from '../outboundcomms/rest';
+import { OutboundMessageProviderService } from './lib/outbound';
+
+const outboundMessageProvider = new OutboundMessageProviderService();
```

Comment:
Instead of instantiating here, let the lib/outbound file to export this instantiation so apps engine can reuse it